internal/pkg/service/buffer/store/stats.go

```diff
@@ -2,22 +2,52 @@ package store
 
 import (
 	"context"
+	"sync"
 
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
 	"github.com/keboola/keboola-as-code/internal/pkg/telemetry"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
+const maxStatsPerTxn = 50
+
 func (s *Store) UpdateSliceStats(ctx context.Context, nodeID string, stats []model.SliceStats) (err error) {
-	_, span := s.tracer.Start(ctx, "keboola.go.buffer.store.UpdateStats")
```

Comment:
Naraz aktualizujeme statistiky pre viacere slices v jednej transakcii.
Transakcia moze mat ale maximalne `128` poloziek, preto sa to musi podelit (po `50`).